packages/CollabCodeEdit-Tests.package/CCEPackageManagerTests.class/instance/testHashchangesAfterMetaClassInformationChange.st

```diff
@@ -2,18 +2,13 @@ tests
 testHashchangesAfterMetaClassInformationChange
 	| originalComment  testClass hashBeforeChange hashAfterChange |
 
-	CCESession default scope: #( 'Signals' ).
-	
-	testClass := (PackageInfo named: 'Signals') classes first.
-	
+	CCESession default scope: #( 'CollabCodeEdit-Tests' ).
+	testClass := Environment default classNamed: 'CCETestDummy'.
 	originalComment := testClass comment.
- 
-	hashBeforeChange := packageManager hashPackages: #( 'Signals' ).
+	[ hashBeforeChange := packageManager hashPackages: #( 'CollabCodeEdit-Tests' ).
```

Comment:
Code following a multiline block being opened should be indented. (See lines below.)